packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -477,20 +495,37 @@ export class CeramicCliUtils {
       // TODO handle invalid configuration
     }
 
-    return await this._saveCliConfig({})
+    await this._saveConfig({}, DEFAULT_CLI_CONFIG_FILE)
+    return {}
+  }
+
+  /**
+   * Load configuration file for the Ceramic Daemon.
+   * @private
+   */
+  static async _loadDaemonConfig(): Promise<DaemonConfig> {
```

Comment:
I guess, it is the next iteration to make it `ConfigFile.load`.

Reply:
I actually think it's generally good practice to separate data storage objects from logic to load said data.  I could see a `ConfigFileLoader.loadConfig()` if we wanted to put the load logic into a class (so that it could be mocked, for example), but I wouldn't want to actually change `DaemonConfig` to a class and put logic into it.  In any case I feel like the simple thing that is done here is probably fine for now.  If we need to be able to mock the load behavior in the future we can clean it up then.

Reply:
Actually, with javascript it would be possible to mock this behavior even with it just being a static function like it is here.  So I'm not really sure what having a separate class for encapsulating the load logic would gain us?

Reply:
> it's generally good practice to separate data storage objects from logic to load said data.

Indeed, and it does not involve `DaemonConfig` interface in any way. `ConfigFile` is to read a file, and provide proper safe accessors like `get('resolvers.nft-did-resolver')`, so that a reader does not do `resolvers?.nft-did-resolver?.whatever` on raw json.

Reply:
ah okay, yeah that makes sense.  Seems not that important at the moment though?

Reply:
Not atm.